src/bin/languageserver/mod.rs

```diff
@@ -280,65 +240,74 @@ impl SolangServer {
                 ..
             } => {
                 if let Some(exp) = cond {
-                    SolangServer::construct_expr(exp, lookup_tbl, symtab, ns);
+                    self.expression(exp, symtab);
                 }
                 for stat in init {
-                    SolangServer::construct_stmt(stat, lookup_tbl, symtab, ns);
+                    self.statement(stat, symtab);
                 }
                 for stat in next {
-                    SolangServer::construct_stmt(stat, lookup_tbl, symtab, ns);
+                    self.statement(stat, symtab);
                 }
                 for stat in body {
-                    SolangServer::construct_stmt(stat, lookup_tbl, symtab, ns);
+                    self.statement(stat, symtab);
                 }
             }
             ast::Statement::DoWhile(_locs, _blval, stat1, expr) => {
-                SolangServer::construct_expr(expr, lookup_tbl, symtab, ns);
+                self.expression(expr, symtab);
                 for st1 in stat1 {
-                    SolangServer::construct_stmt(st1, lookup_tbl, symtab, ns);
+                    self.statement(st1, symtab);
                 }
             }
             ast::Statement::Expression(_locs, _, expr) => {
-                SolangServer::construct_expr(expr, lookup_tbl, symtab, ns);
+                self.expression(expr, symtab);
             }
-            ast::Statement::Delete(_locs, _typ, expr) => {
-                SolangServer::construct_expr(expr, lookup_tbl, symtab, ns);
+            ast::Statement::Delete(_, _, expr) => {
+                self.expression(expr, symtab);
             }
-            ast::Statement::Destructure(_locs, _vecdestrfield, expr) => {
-                SolangServer::construct_expr(expr, lookup_tbl, symtab, ns);
+            ast::Statement::Destructure(_, _vecdestrfield, expr) => {
```

Comment:
Why are these variable names prefixed with an underscore?

Reply:
This is what the hyperledger mentee wrote many years ago. I have just removed a few more of these, I think that's all of them.